app/javascript/components/teams/team-responsibilities.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div
+    v-if="!this.loading"
+    class="flex flex-col space-y-3"
+  >
+    <div class="flex flex-row justify-between w-full">
+      <span class="text-lg font-medium">{{ $t('teams.responsibilities.title') }}</span>
+      <div class="flex flex-row space-x-2">
+        <btn
+          :hierarchy="'secondary'"
+          :icon-file-name="'plus.svg'"
+          :icon-only="true"
+          :icon-classes="'w-4 h-4'"
```

Comment:
Acá hay varios `v-bind` que te puedes ahorrar tb!

```suggestion
          hierarchy="secondary"
          icon-file-name="plus.svg"
          :icon-only="true"
          icon-classes="w-4 h-4"
```